ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG BIGDL_IMAGE_NAME
+ARG BIGDL_IMAGE_VERSION
+
+# stage 1. generate SGX secrets and prepare KMS env
+FROM $BIGDL_IMAGE_NAME:$BIGDL_IMAGE_VERSION as temp
+
+ARG LOCAL_IP
```

Comment:
need to rm LOCAL_IP, may set it in the env, please also test spark local mode.

Reply:
removed and passed spark local test.